ui/src/components/table/CustomTable.tsx

```diff
@@ -150,11 +176,12 @@ function CustomTable({
 
     const generateModalDialog = () => {
         if (entityModal.open) {
-            let label;
+            let label: string | undefined;
+            let services;
```

Comment:
It is recommended to declare a variable in least possible scope.

Reply:
done